config/services.yaml

```
@@ -6,6 +6,78 @@
+          'leadSource': 'asc',
+          'type.value': 'asc',
+          'status.value': 'asc',
+        },
+      ]
+    tags:
+      - { name: 'api_platform.filter', id: 'customer.mongodb.order' }
+
+  app.customer.mongodb.range_filter:
+    parent: 'api_platform.doctrine_mongodb.odm.range_filter'
+    class: App\Shared\Infrastructure\Filter\UlidRangeFilter
+    arguments: [{ 'ulid': ~ }]
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)